django/core/checks/settings_deprecation.py

```diff
@@ -0,0 +1,107 @@
+from django.conf import settings
+from django.core import checks
+
+
+def deprecated_settings_variables(*args, **kwargs):
```

Comment:
I'm not sure if args-kwargs should stay in the function definition when not being used?

Reply:
I thought it's a thing from the system check framework. But if not, I'll gladly remove them.

Reply:
It's important to make an informed decision rather than just taking my word for it. Before writing the check function, you should have investigated why check functions use *args and **kwargs, as this would provide a clearer understanding of how things work in the checks framework.

By the way, I looked into it, and ~*args and~ `**kwargs` are necessary because of [how checks are registered](https://github.com/django/django/blob/c334c1a8ff4579cdb1dd77cce8da747070ac9fc4/django/core/checks/registry.py#L33C1-L50C2). (I didn't investigate further than that.)

Which also brings us to the issue that your check is not registered. While the test passes because the check function is directly invoked, the check will not actually run when other system checks are run. (Since the test should technically be failing, that means your test needs improvement.)